client/components/AddNew.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { Form } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+
+export default class AddNew extends Component {
+  state = {}
+
+  handleChange = (e, { value }) => this.setState({ value })
+
+  render() {
+    const { value } = this.state
+    return (
+      <Form>
+        <Form.Group widths='equal'>
+          <Form.Input fluid label='Name' placeholder='Name' />
+          <Form.Input fluid label='Threat' placeholder='Threat' />
```

Comment:
Nice use of Semantic UI!

Reply:
However, these aren't hooked up to the state yet. To do that, we'd use something like:
```
<Form.Input fluid label='Threat' placeholder='Threat' onChange={this.handleChange} value={this.state.threat} />
```

Reply:
Hooked up to the state with handlechange.